flatland_plugins/src/diff_drive.cpp

```diff
@@ -61,6 +61,8 @@ void DiffDrive::TwistCallback(const geometry_msgs::Twist& msg) {
 
 void DiffDrive::OnInitialize(const YAML::Node& config) {
   YamlReader reader(config);
+  pub_odom_ = reader.Get<bool>("pub_odom", true);
```

Comment:
maybe "enable_odom_pub" and "enable_twist_pub" for these to reduce confusion with the existing variable names.